CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # CHANGELOG
 
+## 5.0.0 (2023-04-11)
```

Comment:
Small nit: we don't add the release date generally until we do the release. We can just update it on release this time though.